include/grpcpp/impl/codegen/completion_queue.h

```diff
@@ -361,7 +367,12 @@ class CompletionQueue : private GrpcLibraryCodegen {
     gpr_atm_no_barrier_fetch_add(&avalanches_in_flight_,
                                  static_cast<gpr_atm>(1));
   }
-  void CompleteAvalanching();
+  void CompleteAvalanching() {
+    if (gpr_atm_no_barrier_fetch_add(&avalanches_in_flight_,
```

Comment:
I'm honestly a little surprised that this has ever worked since I would have expected that calls to `gpr_atm` from `impl/codegen` should have needed `g_core_codegen_interface` , but I see that we were already doing this above (which I added)

Reply:
these are macros.. so they work